packages/cerebral/src/utils.js

```diff
@@ -107,3 +111,19 @@ export function debounce (func, wait) {
 }
 
 export const noop = () => {}
+
+export const forceSerializable = (payload) => {
+  return Object.keys(payload).reduce((currentPayload, key) => {
+    currentPayload[key] = payload[key]
+
+    if (!isSerializable(currentPayload[key])) {
+      Object.defineProperty(currentPayload[key], 'toJSON', {
+        value () {
+          return '[NON_SERIALIZABLE]'
```

Comment:
It is only created once? :) It happens before signal runs... or do I misunderstand?